### PR TITLE
Update settings.php

### PR DESCRIPTION
**Pull request by sherakama**
# Ready for review

Change drupal_exit() to exit() to prevent log flooding of an undefined function.

### CODE REVIEW

File: soe/src/sites/default/settings.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
   header('HTTP/1.0 301 Moved Permanently');
   $redirect_url = "https://" . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
   header("Location: " . check_plain($redirect_url));
-  drupal_exit();
+  exit();
```
Comment (jbickar) [inline soe/src/sites/default/settings.php:12]: Hm. We have this code in a lot of places, methinks. Lemme look.

Comment (jbickar) [reply soe/src/sites/default/settings.php:12]: Like [here](https://github.com/SU-SWS/acsf-cardinald7/blob/1.x/factory-hooks/pre-settings-php/force-https.php#L15)

Comment (sherakama) [reply soe/src/sites/default/settings.php:12]: Yeah, we do. It shows up :alot: in the logs.

Comment (jbickar) [reply soe/src/sites/default/settings.php:12]: Is that the "undefined function `cache_get()`" message?

Comment (sherakama) [reply soe/src/sites/default/settings.php:12]: No, that one has something to do with 404 pages. Acquia connector pings the site endpoint often causing it to happen, but also, normal traffic, seems to cause it.

Comment (sherakama) [reply soe/src/sites/default/settings.php:12]: Acquia connector module is disabled AFAICT

**Review by imonroe — APPROVED** [review]
This looks just fine to me.

**Pull request merged**